integration/kubernetes/confidential/agent_image.bats

```diff
@@ -58,16 +88,6 @@ setup() {
 	if [ -n "$https_proxy" ]; then
 		echo "Enable agent https proxy"
 		add_kernel_params "agent.https_proxy=$https_proxy"
-
-		local local_dns=$(grep nameserver /etc/resolv.conf \
```

Comment:
Do we still need this proxy code for the TDX tests, or is there a new way to do it?

Reply:
For test cases launched by kubernetes, the default etcd service will handle DNS resolve, then this is not need anymore.

Reply:
Great - thanks for the info Arron!

Reply:
Cool! so I will remove it completely from the kubernetes tests.

Reply:
Hi @wainersm @stevenhorsman, sorry for confusing, the DNS part can be removed but for kernel param `agent.https_proxy` is still needed since kata-agent in guest can not get the proxy info.

Reply:
The reason we delete this part but TDX CI still passed is due to https://github.com/kata-containers/kata-containers/blob/CCv0/docs/how-to/data/confidential-agent-config.toml.in#L7 , both test cases will update this part which will set the `https_proxy` through agent config file.

Reply:
ah, got it @arronwy ! Can I merge it then?

Reply:
sure, thanks @wainersm !